### PR TITLE
New package: tecwindow.SoundTranscriber version 1.1.2

### PR DESCRIPTION
**Pull request by MesterPerfect**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/113722&drop=dogfoodAlpha

### CODE REVIEW

**Comment by MesterPerfect** [comment]
@microsoft-github-policy-service agree

**Comment by MesterPerfect** [comment]
Template: msftbot/requiresApproval/moderator

**Comment by MesterPerfect** [comment]
msftbot/requiresApproval/moderator

**Comment by stephengillie** [comment]
Hi @MesterPerfect,

In a VM, this package installed normally, but gave this error when launched:
image

It might be due to the VM not having an audio output device. Is this expected?

File: manifests/t/tecwindow/SoundTranscriber/1.1.2/tecwindow.SoundTranscriber.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: tecwindow.SoundTranscriber
+PackageVersion: 1.1.2
+Installers:
```
Comment (stephengillie) [inline manifests/t/tecwindow/SoundTranscriber/1.1.2/tecwindow.SoundTranscriber.installer.yaml:6]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Comment by stephengillie** [comment]
https://github.com/microsoft/winget-pkgs/issues/111634

**Comment by MesterPerfect** [comment]
Template: msftbot/requiresApproval/moderator

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by MesterPerfect** [comment]
@ItzLevvie

What is supposed to happen now

**Comment by BrandonWanHuanSheng** [comment]
>What is supposed to happen now


It look like the application need dependcies to install successful

**Comment by MesterPerfect** [comment]
> > What is supposed to happen now
> 
> It look like the application need dependcies to install successful

Well, can I add dependencies in winget code or should I add them to the installer

**Comment by BrandonWanHuanSheng** [comment]
> > > What is supposed to happen now
> > 
> > It look like the application need dependcies to install successful
> 
> Well, can I add dependencies in winget code or should I add them to the installer

It look like winget will support package dependcies in new version.

**Comment by MesterPerfect** [comment]
@stephengillie

I have added this to the installer.yaml file

```
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Is this enough to install and run the program without the error message؟

**Comment by stephengillie** [comment]
> > > What is supposed to happen now
> > 
> > 
> > It look like the application need dependcies to install successful
> 
> Well, can I add dependencies in winget code or should I add them to the installer

Adding the dependencies to the installer will allow you to control the install and verify that they're installed. Also it will make the installer more portable and less-dependent on external manifests for any package manager. While we have dependencies enabled in the pipelines to facilitate "grandfathered" packages, we're working to make dependency support more robust before accepting new packages with dependencies.

**Comment by stephengillie** [comment]
> @stephengillie
> 
> I have added this to the installer.yaml file
> 
> ```
> Dependencies:
>   PackageDependencies:
>     - PackageIdentifier: Microsoft.VCRedist.2015+.x64
> ```
> 
> Is this enough to install and run the program without the error message؟

This syntax appears correct.

**Comment by stephengillie** [comment]
Please ignore the `Internal-Error-Dynamic-Scan` error - it's a catch-all error for several pipeline issues we're working to diagnose, and doesn't indicate anything about the package.

**Pull request closed without merging**